src/components/ContactsApp.jsx

```diff
@@ -1,41 +1,27 @@
 import React from "react";
 import ContactsList from "./ContactsList/ContactsList";
 import ContactsHeader from "./ContactsList/ContactsHeader";
-import ContactCard from "./ContactCard/ContactCard";
-import Modal, { ModalContent } from "cozy-ui/react/Modal";
-import { Button } from "cozy-ui/react/Button";
 import { PropTypes } from "prop-types";
-import { Icon, Menu, MenuItem } from "cozy-ui/react";
+import ContactsIntentButton from "./Buttons/ContactsIntentButton";
+import ContactCardModal from "./Modals/ContactCardModal";
 
-const TranslatedContactCard = ({ ...props }, { t }) => (
-  <ContactCard title={t("contact_info")} {...props} />
+const ContactsHeaderWithActions = () => (
+  <ContactsHeader
+    renderActions={() => {
+      const fakeintent = new URL(window.location).searchParams.get(
+        "fakeintent"
+      );
+      return (
+        <div className="actions">
+          {fakeintent !== null && (
+            <ContactsIntentButton>{"Select a Contact"}</ContactsIntentButton>
+          )}
+        </div>
+      );
+    }}
+  />
 );
 
-const ContactCardMenu = ({ deleteAction }) => (
```

Comment:
I move it to its own file https://github.com/cozy/cozy-contacts/pull/48/commits/12af99629749f7770df727c1248895c0b5fc3792#diff-36c3351d71a590cb0fda9f3033e9270f